src/components/WebexMember/WebexMember.jsx

```diff
@@ -27,12 +28,21 @@ export default function WebexMember({
     .find((itemMember) => itemMember.personID === personID);
 
   const isMuted = member && member.muted;
+  const isSharing = member && member.sharing;
 
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
-      {isMuted && <div className="member-muted"><span className="icon icon-microphone-muted_24" /></div>}
+      <div>
+        <div className="member-name">{`${firstName} ${lastName}`}</div>
+        <div className="member-presenter">{isSharing && <span>Presenter</span>}</div>
+      </div>
+      {isSharing && 
+          <div className="member-sharing"><Icon name="icon-content-share_16" /></div>
+      }
+      {isMuted && 
+        <div className="member-muted"><span className="icon icon-microphone-muted_24" /></div>
```

Comment:
Can you please update the icon to use `<Icon />` component as in the line `41` ?
```suggestion
        <div className="member-muted"><Icon name=" icon-microphone-muted_24" /></div>
```